local-cli/server/util/debugger.html

```diff
@@ -248,7 +248,7 @@
     <p>
       React Native JS code runs as a web worker inside this tab.
     </p>
-    <p>Press <kbd id="shortcut" class="shortcut">⌘⌥J</kbd> to open Developer Tools. Enable <a href="https://stackoverflow.com/a/17324511/232122" target="_blank">Pause On Caught Exceptions</a> for a better debugging experience.</p>
+    <p>Press <kbd id="shortcut" class="shortcut">⌘⌥I</kbd> to open Developer Tools. Enable <a href="https://stackoverflow.com/a/17324511/232122" target="_blank">Pause On Caught Exceptions</a> for a better debugging experience.</p>
```

Comment:
I'm pretty sure this is the source. However I've seen the incorrect shortcut in a markdown file somewhere else too, so I will update it there also.